core/spec.md

```diff
@@ -6610,20 +6612,28 @@ Where:
     - Wildcards (`*`) (see below) MUST NOT be present in the `<VALUE>`.
     - MUST only be used on numeric or string attribute types.
 
-When comparing an `<ATTRIBUTE>` to the specified `<VALUE>`, the type of the
-attribute impacts how the comparisons are done:
-- For boolean attributes, a successful `=` comparison MUST be an exact
-  case-sensitive match (`true` or `false`).
-- For numeric attributes, standard numeric comparisons rules apply.
-- For string attributes, the strings MUST be compared in a case-insensitive
-  manner. `<VALUE>` MAY include one or more wildcard (`*`) characters. The
-  presence of a wildcard indicates that any number of characters can appear at
-  that location in the `<VALUE>`. The wildcard MAY be escaped via the use of a
-  backslash (`\\`) character (e.g. `abc\*def`) to mean that the `*` is to be
-  interpreted as a normal character and not as a wildcard. Note that a
-  `<VALUE>` of `*` MUST be equivalent to checking for the existence of the
-  attribute, with any value (even an empty string). In other words, the filter
-  will only fail if the attribute has no value at all.
+For comparing an `<ATTRIBUTE>` to the specified `<VALUE>`, and for purposes
+of sorting (see the [`?sort`](#sort-flag) flag), the type of the attribute
+impacts how the comparisons are done:
+- For boolean attributes, comparisons MUST be an exact case-sensitive match
+  (`true` or `false`). For relative comparisons, `false` is less than `true`.
+- For numeric attributes, standard numeric comparisons rules apply. Note that
```

Comment:
Do we support floating point types? If so, we should probably document NaN comparisons (for both equality and relative ordering).

Reply:
good catch - will do in a follow-on PR
https://github.com/xregistry/spec/issues/361